setup.cfg

```diff
@@ -186,9 +186,7 @@ lint =
     # TODO: consider hosting our own plugin on pypi
     #       or get an equivalent rule added to bandit
     #       for now we pin to a specific commit for safety
-    flake8-markupsafe@git+https://github.com/vmagamedov/flake8-markupsafe#egg=b391bd13df9330e01666d304b7f4403d67e5ceba
     flake8-type-checking
-    ruff<=0.7.1
```

Comment:
This is why you're getting CI errors, ruff is still a dependency, it just shouldn't be pinned to `<=0.7.1`. I'd revert to the previous `!=0.7.2`, since we know that one is broken.

Also make sure to update the pin in `.pre-commit-config.toml` (`make update` or `pre-commit autoupdate` should do that for you)

Reply:
Oh my!!!